cmake/LYPython.cmake

```diff
@@ -93,6 +93,7 @@ function(update_pip_requirements requirements_file_path unique_name)
 
     if (NOT ${PIP_RESULT} EQUAL 0)
         message(CHECK_FAIL "Failed to fetch / update python dependencies: ${PIP_OUT} - use CMAKE_MESSAGE_LOG_LEVEL to VERBOSE for more information")
+        message(FATAL_ERROR "some python dependencies failed to install/download.  This will likely cause errors further down the line, stopping.")
```

Comment:
is it possible to know which dependencies failed? is there a log where someone could go and inspect? can we point them there in this error message?

Reply:
Yes, the line above it shows the full log from running PIP which reveal's the error. ( ${PIP_OUT} )

Reply:
I'll update the message to try to make it clearer.